pkg/pki/x509/e2e.go

```diff
@@ -16,7 +16,7 @@
 //go:build e2e
```

Comment:
I think these test keys and functions need to live in a separate package (`test`) or not be exported. I'd worry about these getting used otherwise.

Reply:
They still need a `rekor` server running.

Reply:
Oh yep, sorry, I see that now.

Still, I don't love exporting static key values. A better fix may be to generate these in memory, but this should be a follow up.

Reply:
>Still, I don't love exporting static key values. A better fix may be to generate these in memory, but this should be a follow up.

I agree. It can be a follow-up PR. Thanks